src/styles/pages/inventory.scss

```diff
@@ -0,0 +1,63 @@
+@use '../variables/colors.scss';
+@use '../variables/fonts.scss';
+@use '../common/common.scss';
+
+.createProduct {
```

Comment:
the naming convention should be consistent, i think we just use hyphen separated for all css classes? so like create-product and create-product-box instead of mixing the 2